uiowa/uiowa
# i3010 - Drawer menu getting cut off on shorter screen height

Closes #3010 


# How to test


1. blt ds --site=grad.uiowa.edu
2. blt frontend
3. Open the menu and expand one of the expandable sections. Then scale the window down as small as possible and make sure you can scroll up and down and that all menu links are visible. Expand children as well to double check.
4. Test in chrome and firefox
